.gitignore

```diff
@@ -1,3 +1,8 @@
+# Editors
+.idea/
```

Comment:
Please put this in a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)